server/core/src/logical_locking.cpp

```diff
@@ -0,0 +1,304 @@
+#include "irods_log.hpp"
+#include "logical_locking.hpp"
```

Comment:
This is more of a nitpick, but I'd recommend always including the header that is being implemented by the cpp file first. While it doesn't necessarily change anything, I think it helps establish relationships between headers and impl files better.

Again, it is totally up to you whether you make this change or not :)

Reply:
Seems reasonable to me. Done